sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -34,7 +36,7 @@
 
     @Override
     protected void beforeTest() {
-        final String queueName = getQueueName();
+        final String queueName = "hemant-test2"; //getQueueName();
```

Comment:
getQueueName(). When running the test if you set the right environment variables, you don't need to make these manual changes.